test/utils.d

```diff
@@ -109,17 +111,19 @@ auto payloadServer(scope HTTPServerRequest req, scope HTTPServerResponse res)
     }
     else
     {
-        scope(failure) {
-            writeln("Remaining expected URLs:", apiExpectations.map!(x => x.url));
-        }
-        assert(0, "Request for unexpected URL received: " ~ req.requestURL);
+        logError("Remaining expected URLs: %s", apiExpectations.map!(x => x.url));
+        logError("Request for unexpected URL received: '%s'", req.requestURL);
+        assert(0);
     }
 
     res.statusCode = expectation.respStatusCode;
     // set failure status code exception to suppress false errors
-    import dlangbot.utils : _expectedStatusCode;
-    if (expectation.respStatusCode / 100 != 2)
-        _expectedStatusCode = expectation.respStatusCode;
+    version (unittest)
```

Comment:
Do you still need this `version(unittest)` now that you fixed your dub build?
It's in the `test` folder (which for dub implies it's a `unittest` build).